.github/workflows/test.yml

```diff
@@ -47,3 +47,33 @@ jobs:
       run: tox -e ${{ matrix.toxenv }} --notest
     - name: test
       run: tox -e ${{ matrix.toxenv }} --skip-pkg-install
+
+  stubgenc:
+    # Check stub file generation for a small pybind11 project
+    # (full text match is required to pass)
```

Comment:
I'm pretty new to gh-actions, and I'm not sure if it's the right place for the check.